test/features/standup.feature

```diff
@@ -0,0 +1,53 @@
+Feature: Standup-module
+
+    Scenario: get list of standups
+        Given make a GET request to "/standup"
+        Then the response status code should be 200
+        And the response should be "[]"
+
+    Scenario: add a new standup
+        Given make a POST request to "/standup" with:
+            | discordUser      | "eddiehubber"          |
+            | yesterdayMessage | "yesterday I did this" |
+            | todayMessage     | "Today I'll do this"   |
+        Then the response status code should be 201
+        And the response should contains:
+            | discordUser      | "eddiehubber"              |
+            | yesterdayMessage | "yesterday I did this"     |
+            | todayMessage     | "Today I'll do this"       |
+            | createdOn        | "2021-01-01T00:00:00.000Z" |
+            | id               | 666                        |
+
+
+    Scenario: search Standup
+        Given make a POST request to "/standup" with:
+            | discordUser      | "eddiehubber"          |
+            | yesterdayMessage | "yesterday I did this" |
+            | todayMessage     | "Today I'll do this"   |
+        Then  make a POST request to "/standup/search" with:
```

Comment:
I think `search` should be a `GET` with a query parameter, something like `GET /standup?discordUser=eddiehubber`

Reply:
I had the same thoughts while coding this. I made it a Post in case the SearchContext is getting more complex in the future so the URL doesnt get too long with too many parameters. 
Do we need more properties be searchable in this context when I adjust this endpoint?

Reply:
I think keep it simple for now.

In the future we might want
- keyword search that searches `yesterdayMessage` and `todayMessage`
- date range filter (for example list the last 3 days for all users, or filter by user as well)

Reply:
Okay I'll implement the simple way for now then! :D